src/components/legacy/markdown/custom-components.tsx

```diff
@@ -228,6 +229,20 @@ const customSatellytesComponents = {
     const { children, ...rest } = props;
     return <Figcaption {...rest}>{children}</Figcaption>;
   },
+  stats(props) {
+    const { statItems } = props;
+    return (
+      <Stats>
+        {statItems.map((item, index) => (
+          <StatItem
+            key={item.label + index}
+            title={item.label}
+            content={item.value}
+          />
+        ))}
+      </Stats>
+    );
+  },
```

Comment:
I think we can move the rendering part directly into `src/components/pages/blog-post/blog-post.tsx` since the properties of `customComponents` represent html tags that are rendered by the markdown plugin.

Reply:
you are right, I fixed it 👍